readme.md

```diff
@@ -0,0 +1,8 @@
+Pełna treść zadania:
```

Comment:
Try to add some styling to readme next time so you can get familiar with markdown format :)

Reply:
For example you can make class names **bold** and method names _italic_